src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.json

```diff
@@ -5093,6 +5278,17 @@
                               "type" : "NamedTypeSpecifier"
                            }
                         }
+                     }, {
+                        "name" : "Quantity",
+                        "value" : {
+                           "path" : "quantity",
+                           "type" : "Property",
+                           "source" : {
+                              "path" : "dispenseRequest",
+                              "scope" : "O",
+                              "type" : "Property"
+                           }
+                        }
                      }, {
```

Comment:
is it possible to add the additional field (e.g. quantity) to the Test patient json:
https://github.com/uwcirg/AHRQ-CDS-Connect-PAIN-MANAGEMENT-SUMMARY/blob/master/src/utils/r4_test_patients/brenda_jackson.json#L757
so maybe we can test out the changes via smarthealthit launcher in dev instance...Just a thought about testing these changes..

Reply:
I did post some dummy MeddicationRequest data for brenda_jason (note, in **R4**) to test out